src/ops.rs

```diff
@@ -201,3 +201,48 @@ impl_op_unary! { impl Neg, neg for  Matrix<T, M, N> }
 impl_op_unary! { impl Neg, neg for &Matrix<T, M, N> }
 impl_op_unary! { impl Not, not for  Matrix<T, M, N> }
 impl_op_unary! { impl Not, not for &Matrix<T, M, N> }
+
+////////////////////////////////////////////////////////////////////////////////
+// Matrix * Matrix
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
Could you put this new section just below the `Matrix + Matrix` section. I think it makes sense to put it there. Same for in the tests module.